spec/features/atom/users_spec.rb

```diff
@@ -4,17 +4,19 @@
   describe "GET /" do
     let!(:user) { create(:user) }
 
-    context "user atom feed via private token" do
+    context 'user atom feed via private token' do
       it "should render user atom feed" do
         visit user_path(user, :atom, private_token: user.private_token)
-        body.should have_selector("feed title")
+        expect(body).to have_selector('feed title')
       end
     end
 
     context 'feed content' do
       let(:project) { create(:project) }
-      let(:issue) { create(:issue, project: project, author: user, description: '') }
-      let(:note) { create(:note, noteable: issue, author: user, note: 'Bug confirmed', project: project) }
+      let(:issue) { create(:issue, project: project,
+                           author: user, description: '') }
+      let(:note) { create(:note, noteable: issue, author: user,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.